internal/guest/runtime/runc/utils.go

```diff
@@ -101,6 +110,34 @@ type standardLogEntry struct {
 	Err     error        `json:"error,omitempty"`
 }
 
+func (l *standardLogEntry) asError() (err error) {
```

Comment:
I'm not the biggest fan of this function, trying to see if I can think of another way of doing things

Reply:
Why is wrapping the string an issue?

Reply:
Can we not directly reference the errors in https://github.com/opencontainers/runc/blob/master/libcontainer/error.go instead of re-hardcoding the strings here?

Reply:
For the string wrapping, that means elsewhere in the code, if we want to [match against the error returned](https://github.com/microsoft/hcsshim/pull/1272/files#diff-5d9153b1eb1210c5b2c5589fa471d062c81417e39090309289c260a05b1eb248R262), we'd have to use string search, and that would leak into the [runc handling code](https://github.com/microsoft/hcsshim/pull/1272/files#diff-5d9153b1eb1210c5b2c5589fa471d062c81417e39090309289c260a05b1eb248L168-L173)